### PR TITLE
Release pull request for Hypothesis 3.6.1

### PR DESCRIPTION
**Pull request by DRMacIver**
Provides change log and version bump and nothing else, but I need to do a pull request because protected branches. :-)

### CODE REVIEW

File: docs/changes.rst [hunk 1]
```diff
@@ -21,6 +21,30 @@ Hypothesis APIs come in three flavours:
 You should generally assume that an API is internal unless you have specific
 information to the contrary.
 
+
+------------------
+3.6.1 - 2016-12-20
+------------------
+
+This release fixes a dependency problem and makes some small behind the scenes
+imporvements.
```
Comment (alexwlchan) [inline docs/changes.rst]: /s/imporvements/improvements

File: docs/changes.rst [hunk 2]
```diff
@@ -21,6 +21,30 @@ Hypothesis APIs come in three flavours:
...
+
+* The fake-factory dependency was renamed to faker. If you were depending on
```
Comment (alexwlchan) [inline docs/changes.rst]: Re-reading: I think there’s the potential for confusion here. Should I switch to using `pip install hypothesis[faker]`? Or stick with `pip install hypothesis[fake-factory]`? Or will either work?

Long-term I think we should switch to `hypothesis[faker]` for consistency, not necessarily now – but we should be clear whether that changes right now.

**Comment by DRMacIver** [comment]
It was easy enough to do, so I've updated it so that either hypothesis[faker] or hypothesis[fakefactory] will work.

Long term I think I'd actually like to get rid of the hypothesis[*] extras altogether and instead have people manage their dependencies separately from Hypothesis (no reason to actually remove them for now, but at least they can be removed from documentation). The main limitation on doing that right now is that the django one still depends on pytz, and this plan only works if each extra module is single-dependency.

**Comment by alexwlchan** [comment]
Finally a green run on CI! Merging. Over to you to actually stick the release on PyPI. 😄

**Pull request merged**